analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -139,6 +139,12 @@ public class SymbolReferenceAnalyzerTest
             // This path is unreachable for VB code
             new TestSymbolReferenceAnalyzer_VB(null, isTestProject).TestGetSetKeyword(null).Should().BeNull();
 
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void Verify_TokenThreshold() =>
+            // In TokenThreshold.cs there are 4009 tokens which is more than the current limit of 40000
```

Comment:
Typo? 
```suggestion
            // In TokenThreshold.cs there are 40009 tokens which is more than the current limit of 40000
```
or
```suggestion
            // In TokenThreshold.cs there are 40090 tokens which is more than the current limit of 40000
```
I don't know

Reply:
😄

It's `40009`